python/ray/autoscaler/_private/util.py

```diff
@@ -384,13 +384,35 @@ def fill_node_type_min_max_workers(config):
                 node_type_data.setdefault("max_workers", global_max_workers)
 
 
+def with_envs(cmds, kv=Dict[str, str]) -> str:
+    """
+    Returns a list of commands with the given environment variables set.
+
+    Args:
+        cmds (List[str]): List of commands to set environment variables for.
+        kv (Dict[str, str]): Dictionary of environment variables to set.
+
+    Returns:
+        List[str]: List of commands with the given environment variables set.
+
+    Example:
+        with_envs(["echo $FOO"], {"FOO": "BAR"})
+            -> ["export FOO=BAR; echo $FOO"]
+    """
+    out_cmds = []
+    for cmd in cmds:
+        kv_str = ""
+        for k, v in kv.items():
+            kv_str += f"export {k}={v}; "
```

Comment:
We don't need export. Something like `ENV=VAR $command` should just work? Isn't export corrupting the follow-up commands?

Reply:
yeah, I agree - it was more of keeping the same with legacy code.